src/openforms/registrations/contrib/objects_api/validators.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext as _
+from django.utils.deconstruct import deconstructible
+from django.template import TemplateSyntaxError
+from openforms.registrations.contrib.objects_api.constants import (
+    JsonTemplateValidatorErrorTypes,
+)
+
+from rest_framework import serializers
+
+from openforms.template import render_from_string, openforms_backend
+
+
+@deconstructible
+class JsonTemplateValidator:
+    """Validate that the converted django template is legal JSON"""
+
+    def __init__(
+        self,
+        max_length: int | None = None,
+        backend: str = openforms_backend,
+        error_type: JsonTemplateValidatorErrorTypes.choices = JsonTemplateValidatorErrorTypes.model,
```

Comment:
The max_length is not needed (see previous comment), and the error type also not.

Also see other comments about the backend